internal/annotators/http/pki.go

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright 2021 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package http
+
+import (
+	"context"
+	"net/http"
+	"os"
+
+	"github.com/project-alvarium/alvarium-sdk-go/internal/annotators"
+	"github.com/project-alvarium/alvarium-sdk-go/pkg/config"
+	"github.com/project-alvarium/alvarium-sdk-go/pkg/contracts"
+	"github.com/project-alvarium/alvarium-sdk-go/pkg/interfaces"
+)
+
+// PkiAnnotator is used to validate whether the signature on a given piece of data is valid
+type HTTPPkiAnnotator struct {
```

Comment:
Note the type name in the comment is incorrect. Also, please rename to `HttpPkiAnnotator`

Reply:
Fixed in ef52a4d46b344e24703744e982ac2fc17aed75c4